src/Microsoft.Health.Dicom.Core/Modules/ServiceModule.cs

```diff
@@ -143,8 +143,6 @@ public void Load(IServiceCollection services)
                    .AsSelf()
                    .AsImplementedInterfaces();
 
-            services.AddApplicationInsightsTelemetry();
-
             services.AddSingleton<BackgroundServiceHealthCheckCache>();
 
             services.AddHealthChecks().AddCheck<BackgroundServiceHealthCheck>(name: "BackgroundServiceHealthCheck");
```

Comment:
I realize this isn't part of this PR, but I would imagine that `services.AddSingleton<BackgroundServiceHealthCheckCache>();` and  `.AddCheck<BackgroundServiceHealthCheck>(name: "BackgroundServiceHealthCheck");` would be part of `AddBackgroundWorkers`. This seems like a cross of concerns.

Reply:
Also, it may be worth seeing if the background stuff fits into our `Module` paradigm.

Reply:
That looks like the appropriate place for this. This background check appears to be only a proxy for the service's health.